net/base/cache_type.h

```diff
@@ -24,9 +24,6 @@ enum BackendType {
   CACHE_BACKEND_DEFAULT,
   CACHE_BACKEND_BLOCKFILE,  // The |BackendImpl|.
   CACHE_BACKEND_SIMPLE,     // The |SimpleBackendImpl|.
-#if defined(STARBOARD)
-  CACHE_BACKEND_COBALT      // The |CobaltBackendImpl|,
```

Comment:
This isn't actually used by anything, the used backend is always DEFAULT. Took the opportunity to clean this up.

Reply:
nice!